app/controllers/api/teams.php

```diff
@@ -358,6 +360,7 @@
                     'email' => $email,
                     'emailVerification' => false,
                     'status' => true,
+                    'passwordHistory' => $passwordHistory > 0 ? [Auth::passwordHash(Auth::passwordGenerator(), Auth::DEFAULT_ALGO, Auth::DEFAULT_ALGO_OPTIONS)] : [],
```

Comment:
Because this is a random password, maybe we should not store it in the history list?

Reply:
Okay

Reply:
done.